### PR TITLE
Add mutation-and-cna-together mode to plots

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

**Comment by adamabeshouse** [comment]
copy number box plot order is messed up

**Comment by adamabeshouse** [comment]
make dots bigger to accomodate mutation and cna

**Comment by adamabeshouse** [comment]
add more domain padding

**Comment by adamabeshouse** [comment]
render priority

File: src/pages/resultsView/plots/PlotsTab.tsx [hunk 1]
```diff
@@ -736,13 +755,7 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
     }
 
     @computed get scatterPlotSymbol() {
```
Comment (inodb) [inline src/pages/resultsView/plots/PlotsTab.tsx]: `@computed` for constant

File: src/pages/resultsView/plots/PlotsTabUtils.tsx [hunk 2]
```diff
@@ -597,23 +605,23 @@ export const mutationSummaryRenderPriority = stringListToIndexSet(mutationSummar
 const cnaToAppearance = {
     "-2":{
         legendLabel: "Deep Deletion",
-        stroke:"#00008B",
+        stroke:CNA_COLOR_HOMDEL,
     },
     "-1":{
         legendLabel: "Shallow Deletion",
-        stroke:"#00BFFF",
+        stroke:"#2aced4",
```
Comment (inodb) [inline src/pages/resultsView/plots/PlotsTabUtils.tsx]: these are slight variations of default coloring?

File: src/pages/resultsView/plots/PlotsTab.tsx (hunk 1 above)
Comment (adamabeshouse) [reply src/pages/resultsView/plots/PlotsTab.tsx]: 👍

File: src/pages/resultsView/plots/PlotsTabUtils.tsx (hunk 2 above)
Comment (adamabeshouse) [reply src/pages/resultsView/plots/PlotsTabUtils.tsx]: they're slight variations (mainly, increased saturation) from the oncoprint colors

Comment (inodb) [reply src/pages/resultsView/plots/PlotsTabUtils.tsx]: @adamabeshouse might be nice to use sass instead, e.g. `saturate( $base-color, 20% )`

**Pull request merged**